tests/lib/rules/semi.js

```diff
@@ -43,6 +43,16 @@ eslintTester.addRuleTest("lib/rules/semi", {
         { code: "for (let thing of {}) {\n  console.log(thing);\n}", ecmaFeatures: { forOf: true, blockBindings: true }}
     ],
     invalid: [
+        { code: "import * as utils from './utils'", ecmaFeatures: { modules: true }, errors: [{ message: "Missing semicolon.", type: "ImportDeclaration"}] },
```

Comment:
Can you also add some invalid tests to be sure we are honoring never?

Also, can you add a few valid tests?